solutions/src/test/java/ru/sberbank/school/task08/KryoSerializableManagerTest.java

```diff
@@ -0,0 +1,59 @@
+package ru.sberbank.school.task08;
+
+import org.junit.Assert;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import org.junit.Before;
+import org.junit.Test;
+import ru.sberbank.school.task08.state.Savable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KryoSerializableManagerTest {
+    private Savable mapState;
+    private KryoSerializableManager manager;
+    private String path;
+    private String fileName;
+
+    public KryoSerializableManagerTest () {
+        path = "./src/test/java/ru/sberbank/school/task08";
+        fileName = "game.bin";
+    }
+
+    @Before
+    public void initialize () {
+        manager = new KryoSerializableManager(path);
+        manager.initialize();
+        List<GameObject> list = new ArrayList<>();
+        list.add((GameObject) manager.createEntity(InstantiatableEntity.Type.BUILDING, InstantiatableEntity.Status.DESPAWNED, 5000L));
+        list.add((GameObject) manager.createEntity(InstantiatableEntity.Type.ENEMY, InstantiatableEntity.Status.SPAWNED, 5000L));
+        list.add((GameObject) manager.createEntity(InstantiatableEntity.Type.ITEM, InstantiatableEntity.Status.SPAWNED, 5000L));
+        list.add((GameObject) manager.createEntity(InstantiatableEntity.Type.NPC, InstantiatableEntity.Status.SPAWNED, 5000L));
+        list.add((GameObject) manager.createEntity(InstantiatableEntity.Type.BUILDING, InstantiatableEntity.Status.KILLED, 5000L));
+        list.add((GameObject) manager.createEntity(InstantiatableEntity.Type.NPC, InstantiatableEntity.Status.DESPAWNED, 5000L));
```

Comment:
Line is longer than 120 characters (found 129).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a536643203>*